gradle/README.md

```diff
@@ -0,0 +1,6 @@
+# Gradle
+
+To regenerate the [project-schema.json](project-schema.json) file run this in the project's root directory:
+```bash
+./gradlew kotlinDslAccessorsSnapshot
```

Comment:
Should it be tracked in source or not? I can't find any documentation on it at all.